### PR TITLE
Doc update

### LINKED ISSUE #443: Add missing documentation for some connectors
Not all connectors are documented https://docs.microstream.one/manual/storage/storage-targets/index.html.

The following, previously enterprise connectors, are missing

- SAP HANA
- Google Firestore
-

### PR DESCRIPTION
**Pull request by fh-ms**
Update some Javadoc.
Add missing documentation for some connectors #443

### CODE REVIEW

**Comment by fh-ms** [comment]
@hg-ms maybe we should tackle #304 in this pr as well

File: docs/modules/storage/pages/storage-targets/blob-stores/google-cloud-firestore.adoc [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+[options="header", cols="1,2a"]
+|===
+|Property   
+|Description   
+//-------------
+|database-id
+|The database ID to use with this Firestore client.
+
+|emulator-host
+|Tets the emulator host to use with this Firestore client.
```
Comment (hg-ms) [inline docs/modules/storage/pages/storage-targets/blob-stores/google-cloud-firestore.adoc]: Typo?

**Review by hg-ms — CHANGES_REQUESTED** [review]

File: docs/modules/storage/pages/storage-targets/blob-stores/google-cloud-firestore.adoc (hunk 1 above)
Comment (fh-ms) [reply docs/modules/storage/pages/storage-targets/blob-stores/google-cloud-firestore.adoc]: Fixed

**Review by hg-ms — APPROVED** [review]

**Review by rdebusscher — APPROVED** [review]
looks good

**Pull request merged**